models/graphunet.py

```diff
@@ -74,12 +75,20 @@ class GraphUNet(nn.Module):
     def __init__(self, in_features=2, out_features=3):
         super().__init__()
 
-        self.A_0 = Parameter(torch.eye(29).float(), requires_grad=True)
-        self.A_1 = Parameter(torch.eye(15).float(), requires_grad=True)
-        self.A_2 = Parameter(torch.eye(7).float(), requires_grad=True)
-        self.A_3 = Parameter(torch.eye(4).float(), requires_grad=True)
-        self.A_4 = Parameter(torch.eye(2).float(), requires_grad=True)
-        self.A_5 = Parameter(torch.eye(1).float(), requires_grad=True)
+        if use_cuda and torch.cuda.is_available():
```

Comment:
This `if` statement is what I was looking for earlier!  BUT since `use_cuda` is always `False` it never gets used.  :/